src/vs/workbench/contrib/terminal/test/browser/terminalStatusList.test.ts

```diff
@@ -11,23 +11,27 @@ import { spinningLoading } from 'vs/platform/theme/common/iconRegistry';
 import { ThemeIcon } from 'vs/base/common/themables';
 import { TerminalStatusList } from 'vs/workbench/contrib/terminal/browser/terminalStatusList';
 import { ITerminalStatus } from 'vs/workbench/contrib/terminal/common/terminal';
+import { ensureNoDisposablesAreLeakedInTestSuite } from 'vs/base/test/common/utils';
+import { DisposableStore } from 'vs/base/common/lifecycle';
 
 function statusesEqual(list: TerminalStatusList, expected: [string, Severity][]) {
 	deepStrictEqual(list.statuses.map(e => [e.id, e.severity]), expected);
 }
 
 suite('Workbench - TerminalStatusList', () => {
+	let store: DisposableStore;
 	let list: TerminalStatusList;
 	let configService: TestConfigurationService;
 
 	setup(() => {
+		store = new DisposableStore();
```

Comment:
Why do some of these files do `store = new DisposableStore();` and other do `store = ensureNoDisposablesAreLeakedInTestSuite();`?

Reply:
Just because I added `ensureNoDisposablesAreLeakedInTestSuite` returning the store mid way through